packages/block-editor/src/components/contrast-checker/index.js

```diff
@@ -11,6 +11,22 @@ import { __ } from '@wordpress/i18n';
 import { Notice } from '@wordpress/components';
 import { useEffect } from '@wordpress/element';
 
+function ContrastCheckerMessage( { tinyBackgroundColor, tinyTextColor, backgroundColor, textColor } ) {
```

Comment:
Out of curiosity, what ESLint rule was triggered which enforced this refactor? I'm wondering what is the lesson learned.

Reply:
The reason is that you can use hooks after "early returns"